eisbuk-admin/src/pages/customer_area/CustomerSlots/CustomerSlots.tsx

```diff
@@ -59,22 +59,20 @@ const CustomerSlots: React.FC<Props> = ({
   const slotDates = Object.keys(slots);
 
   // if `view=book_ice` should order slot days mondays first and so on
-  // if `view=book_off_ice` display week in standard order
+  // if `view=book_off_ice` display in standard order
   const orderedDates =
-    view === CustomerRoute.BookIce ? orderByWeekDay(slotDates) : slotDates;
+    view === CustomerRoute.BookIce
+      ? orderByWeekDay(slotDates)
+      : orderByDate(slotDates);
```

Comment:
ISO dates sort lexycographically, so converting them to `DateTime` using the `fromISO` function is not needed (it's currently done in the `orderByDate` function). `slotDates.slice().sort()` should be enough

Reply:
Why `slice()` though? @silviot